common/src/main/java/org/broadleafcommerce/common/event/BroadleafJobsEventDetail.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * #%L
+ * BroadleafCommerce Common Libraries
+ * %%
+ * Copyright (C) 2009 - 2018 Broadleaf Commerce
+ * %%
+ * Licensed under the Broadleaf Fair Use License Agreement, Version 1.0
+ * (the "Fair Use License" located  at http://license.broadleafcommerce.org/fair_use_license-1.0.txt)
+ * unless the restrictions on use therein are violated and require payment to Broadleaf in which case
+ * the Broadleaf End User License Agreement (EULA), Version 1.1
+ * (the "Commercial License" located at http://license.broadleafcommerce.org/commercial_license-1.1.txt)
+ * shall apply.
+ * 
+ * Alternatively, the Commercial License may be replaced with a mutually agreed upon license (the "Custom License")
+ * between you and Broadleaf Commerce. You may not use this file except in compliance with the applicable license.
+ * #L%
+ */
+package org.broadleafcommerce.common.event;
+
+import java.io.Serializable;
+
+/**
+ * Effectively a copy of com.broadleafcommerce.jobsevents.domain.dto.SystemEventDetailDTO
+ * to be used when creating a org.broadleafcommerce.common.event.BroadleafJobsEvent
+ * 
+ * @author Jay Aisenbrey (cja769)
+ *
+ */
+public class BroadleafJobsEventDetail implements Serializable {
```

Comment:
I originally thought this was completely unnecessary but I think it _is_ worth having the `friendlyName` and `blob` as options. However, both of those fields should be completely optional. I expect that general usage would only set a `value`

Reply:
That's the reason there's different constructors